asciidoc-release-notes/README.adoc

```diff
@@ -0,0 +1,10 @@
+== SUSE Product 1.0 (SP 1.0) Release Notes
+
+== How to contribute
+
+For a comprehensive overview, see https://confluence.suse.com/display/documentation/Contributing+to+release+notes.
+
+1. Create a new branch with name that includes: your username, the original branch name, and descriptive suffix (e.g. `lkucharczyk-15_SP3-newfeature`).
+2. Submit the changes as merge request.
+3. Write one sentence per line.
+4. Include Jira or Bugzilla identifiers either in comment (`//jsc-SLE-1234`) or in the section ID (`[#jsc-SLE-1234]`).
```

Comment:
Add an article again ...

```suggestion
. Include Jira or Bugzilla identifiers either in a comment (`//jsc#SLE-1234`) or in the section ID (`[#jsc-SLE-1234]`).
```

In addition, this is not a very clear description, maybe instead use something like..?

```suggestion
. If there is a related issue, use its Jira or Bugzilla identifier as the section ID (`[#jsc-SLE-1234]`), or if that is not possible, add it as a comment (`// jsc#SLE-1234`).
```